app/api/v1/antenna/[id]/route.ts

```diff
@@ -42,3 +42,57 @@ export async function DELETE(_: Request, context: { params: { id: string } }) {
     return NextResponse.json({ message }, { status });
   }
 }
+export async function PUT(
+  request: Request,
+  context: { params: { id: number } }
+) {
+  try {
+    if (!context.params.id) {
+      throw {
+        status: 500,
```

Comment:
This should throw 400 status error (Bad request). 500 status code errors are reserved for internal server errors (Connection with database fail, etc)